src/scss/videojs-chromecast.scss

```diff
@@ -0,0 +1,77 @@
+// Colors
+$chromecast-color-main: #cccccc !default;
+
+// Icon files
+$icon-chromecast--default: 'images/ic_cast_white_24dp.png' !default;
+$icon-chromecast--hover: 'images/ic_cast_white_24dp.png' !default;
+$icon-chromecast--casting: 'images/ic_cast_connected_white_24dp.png' !default;
+$icon-chromecast--casting-hover: 'images/ic_cast_connected_white_24dp.png' !default;
```

Comment:
This naming seems to suggest there are four states for one icon. Isn't there two icons, `icon-chromecast` and `icon-chromecast-casting`, each with two state, 'default' and `hover`?